File: Business_Card_BackEnd/BusinessCardInformation.API/BusinessCardInformation.Core/Models/Response/BusinessCard.cs
using Microsoft.EntityFrameworkCore;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BusinessCardInformation.Core.Models.Response
{
    public class BusinessCard
    { 
        public int BusinessCardId { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public string Gender { get; set; }

        [Required]
        [DataType(DataType.Date)]
        public DateTime DateOfBirth { get; set; }

        [Required]
        [EmailAddress]
        public string Email { get; set; }

        [Required]
        [Phone]
        public string Phone { get; set; }

        // Optional, limit size in controller
        public byte[]? Photo { get; set; }

        [Required]
        public string Address { get; set; }
    }
}

File: Business_Card_FrontEnd/BusinessCardFrontEnd/src/model/BusinessCard.ts
import { PaginatedResult } from "./paginated.result";

export class BusinessCard {
    businessCardId: number | undefined;
    name: string | undefined;
    gender: string | undefined;
    dateOfBirth: Date | undefined;
    email: string | undefined;
    phone: string | undefined;
    photo: string | undefined;
    address: string | undefined;
  }
  

  export class BusinessCardFilter extends PaginatedResult<BusinessCard> { 
    name: string | undefined; 
    gender: string | undefined;
    dateOfBirth: string | undefined;   
    date: Date | undefined;
    email: string | undefined;
    phone: string | undefined; 
  }
  
  